pkg/block/index.go

```diff
@@ -632,34 +645,48 @@ func rewrite(
 			continue
 		}
 
-		if err := chunkw.WriteChunks(chks...); err != nil {
+		series = append(series, seriesRepair{
+			lset: lset,
+			chks: chks,
+		})
+	}
+
+	if all.Err() != nil {
+		return errors.Wrap(all.Err(), "iterate series")
+	}
+
+	// sort the series -- if labels moved around the ordering will be different
```

Comment:
Let's always keep comments a full sentence.
```suggestion
	// Sort the series. If labels moved around the ordering will be different.
```

Reply:
Fixed.